src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Controls/ListBox.cs

```diff
@@ -350,6 +350,30 @@ protected override void OnKeyDown(KeyEventArgs e)
                 case Key.Down:
                 case Key.Right:
                     {
+                    if ((Keyboard.Modifiers & ModifierKeys.Alt) == ModifierKeys.Alt)
+                        {
+                            if(e.OriginalSource is GridViewColumnHeader gridViewColumnHeader)
+                            {
+                                if (key == Key.Left)
+                                {
+                                    if(gridViewColumnHeader.Column.ActualWidth > 0)
+                                    {
+                                        gridViewColumnHeader.Width = gridViewColumnHeader.Column.ActualWidth - ColumnWidthStepSize;
+                                        gridViewColumnHeader.UpdateColumnHeaderWidth(gridViewColumnHeader.Width);
```

Comment:
This is different from the mouse code, which only calls UpdateColumnHeaderWidth (without setting header.Width directly).  Why?   I'd expect this to do the same.

Reply:
In Mouse we are capturing mouse position and that is giving us new width and in mouse it can increase or decrease by any width, whereas for keyboard we will be resizing by a fixed size whenever correct key combination is hit.